schema/migration-3-0002-20200521.sql

```diff
@@ -9,5 +9,10 @@ CREATE INDEX IF NOT EXISTS idx_tx_block_id ON tx(block_id);
 CREATE INDEX IF NOT EXISTS idx_reward_spendable_epoch ON reward(spendable_epoch);
 CREATE INDEX IF NOT EXISTS idx_epoch_stake_epoch_no ON epoch_stake(epoch_no) ;
 CREATE INDEX IF NOT EXISTS idx_pool_metadata_ref_pool_id ON pool_metadata_ref(pool_id);
-CREATE INDEX IF NOT EXISTS idx_pool_offline_fetch_error_pmr_id ON pool_offline_fetch_error (pmr_id);
-CREATE INDEX IF NOT EXISTS idx_pool_offline_data_pmr_id ON pool_offline_data (pmr_id);
+
+-- as of 13.2.0.0 offline was renamed to off_chain so we rename previous indexes if they exists
+ALTER INDEX [IF EXISTS] idx_pool_offline_data_pmr_id TO idx_off_chain_pool_data_pmr_id;
+ALTER INDEX [IF EXISTS] idx_pool_offline_fetch_error_pmr_id TO idx_off_chain_pool_fetch_error_pmr_id;
+
+CREATE INDEX IF NOT EXISTS idx_off_chain_pool_fetch_error_pmr_id ON off_chain_pool_fetch_error (pmr_id);
+CREATE INDEX IF NOT EXISTS idx_off_chain_pool_data_pmr_id ON off_chain_pool_data (pmr_id);
```

Comment:
@kderme I've amended it so that if the index already exists then it will be renamed otherwise create the index `IF NOT EXISTS` when a new db.

Let me know if this is the right file to do this rename. as I'm unsure as to when this file is ran. (before/after `migrate-2`, when running migration script, on new startup 🤷)

Reply:
I think this is correct now!
Stage-3 runs after stage-2. Stage-3 files are always executed and their Idempotence is guaranteed by the `IF (NOT) EXISTS` clause. While stage-2 idepotence is guaranteed by the `IF next_version =` clause